File: test-data/test_schemas_matching.py
# Test data with intentionally matching schemas for testing

# [agree:match_test:pydantic]
class MatchTestSchema(BaseModel):
    id: int
    name: str
    email: str
    active: bool
# [agree:end]

# [agree:match_test:sqlalchemy]
class MatchTest(Base):
    __tablename__ = "match_tests"
    id = Column(Integer, primary_key=True)
    name = Column(String, nullable=False)
    email = Column(String, unique=True)
    active = Column(Boolean, default=True)
# [agree:end]

File: test-data/test_schemas_matching.ts
// Test data with schemas that should match Python counterparts

// [agree:match_test:zod]
export const MatchTestSchema = z.object({
  id: z.number(),
  name: z.string(),
  email: z.string(),
  active: z.boolean(),
});
export type MatchTest = z.infer<typeof MatchTestSchema>;
// [agree:end]